modules/ROOT/pages/use-anypoint-monitoring.adoc

```diff
@@ -0,0 +1,115 @@
+
+* A backup and restore of Runtime Fabric
+* An upgrade of Runtime Fabric
+* OS patching of Runtime Fabric nodes
+--
+
+== Logs 
+
+Anypoint Monitoring provides access to xref:monitoring::logs.adoc[log data] for applications deployed to Rutime Fabric. To access logs with Anypoint Monitoring, you need a https://www.mulesoft.com/anypoint-pricing[Titanium subscription]. 
+
+Each Mule app and API gateway replica can store up to 450 megabytes of log data on disk, after which log data 
+is rotated to provide storage for more recent log data.
+
+Logs generated by each replica of a Mule app and API gateway are collected and written to disk on the worker 
+node on which they are running. Each worker node runs a single log forwarding agent, which monitors this location 
+and transmits the log data to Anypoint Monitoring.
+
+[NOTE]
+====
+The log forwarding agent can transmit up to 300 kilobytes of log data per second. You should not deploy more than
+50 replicas per worker node.
+
+In some cases, logs with multiple lines might not be fully transmitted to Anypoint Monitoring. This can occur if 
+multiple lines of log output are split across two files due to log rotation inside Runtime Fabric.
+====
+
+You can enable log forwarding and set log levels when you xref:deploy-to-runtime-fabric.adoc#configure-logging-and-log-levels[deploy applications in Runtime Manager]. 
+
+=== Resource and Permission Requirements for Anypoint Monitoring Agent
+
+MuleSoft runs Anypoint Monitoring agents alongside Mule apps to send telemetry to Anypoint control plane.
+
+For Mule apps deployed or managed using Runtime Fabric agent version 2.5.x or lower, two different agents are used as follows:
+
+* One daemonset per underlying node to transporting logs to Anypoint monitoring control plane.
+* One sidecar per application replica to transporting metrics to Anypoint monitoring control plane. 
+
+For optimal daemonset functionality, follow these prerequisites:
+
+. Check that the application containers write logs to the following locations:
+
+* `/var/log/pods`
+* `/var/log/containers`
+* `/var/lib/docker/containers`
+
+. Check the daemon-set creates and writes on the host path `/var/lib/filebeat-data`.
+. Ensure that the Docker daemon config includes the `json-file` logging driver.
+. Ensure that the Anypoint Monitoring log forwarder daemon-set container has write access to the host on the host path `/var/lib/filebeat-data`.
+
+For applications deployed or managed using Runtime Fabric versions 2.6.x and above, MuleSoft is unifying telemetry transport to a single sidecar per application replica.
+
+To accommodate added functionality, this sidecar now requires higher resource allocation, ranging from 0.1CPU (minimum) to 30% of CPU Limit configured for application replica (maximum) to accommodate for needs of the Mule app. Memory consumption may range from 70 MB (minimum) to 150 MB (maximum).
```

Comment:
@kevintroller can we change "To accommodate added functionality, this sidecar now requires higher resource allocation, ranging from 0.1CPU (minimum) to 30% of CPU Limit configured for application replica (maximum) to accommodate for needs of the Mule app. Memory consumption may range from 70 MB (minimum) to 150 MB (maximum)." statement to below:
Dependent on the features used by customers, the sidecar agent consumes corresponding resources for optimal performance:
-  To enable default Anypoint metrics, the sidecar agent (per application replica) requires 0.05vCPU/Core of CPU allocation and 90MB memory allocation
- If customer opts into forwarding application logs to Anypoint Monitoring<link: https://docs.mulesoft.com/runtime-fabric/latest/deploy-to-runtime-fabric#configure-logging-and-log-levels>, the sidecar agent (per application replica) requires upto 30% of CPU Limit configured for application replica and upto 150MB memory allocation